pallets/gear/src/tests.rs

```diff
@@ -12866,23 +12866,94 @@ fn relay_messages() {
     );
 }
 
+#[test]
+fn wasm_data_section_out_of_static_memory() {
```

Comment:
these tests can be placed in `core/src/code/mod.rs`

Reply:
Done